pkg/channel/fanout/fanout_event_handler.go

```diff
@@ -55,6 +55,7 @@ type Subscription struct {
 	Name           string
 	Namespace      string
 	UID            types.UID
+	EventFormat    *kncloudevents.EventFormat
```

Comment:
Let's not include changes here for now, and keep this PR focused only on updating the dispatcher package